apps/cyberstorm-nextjs/utils/provider.tsx

```diff
@@ -11,10 +18,23 @@ function Providers({ children }: React.PropsWithChildren) {
 
   return (
     <QueryClientProvider client={client}>
```

Comment:
I'm out of the loop about what has been discussed during the milestone, and it's a while since I worked with Dapper, but I'm a bit confused why we have a separate QueryProvider? Wasn't one of the points of Dapper that it would handle this stuff, so the consuming apps/components wouldn't need to worry where the data comes from? If they need to set up a query provider, it kinda sounds Dapper no longer fully insulates the from that.

(It might be that I get the point while I read the rest of the PR, I'll try to remember to edit the comment if I do.)